### ISSUE TITLE
Work on Firefox, but don't on Chrome

### ISSUE THREAD

**Issue by altblog** [streamproc/MediaStreamRecorder#107]
Upload MediaStreamRecorder on FTP — http://geeker.ru/files/webcamrec/demos/video-recorder.html
It work (record webcam video) on Firefox 49.0.2 but don't work on Chrome 49.0.2623.112 m.

**Comment by muaz-khan** [comment]
Requires HTTPs on Chrome.

**Comment by webmagnets** [comment]
I'm on https and it still doesn't work in Chrome; only Firefox.

**Comment by muaz-khan** [comment]
Did you check this demo?

* https://www.webrtc-experiment.com/msr/video-recorder.html

**Comment by webmagnets** [comment]
Yes. I see the video of myself, but nothing gets recorded. No file is
created.


-
*Nathan Cain*
501-463-6336
www.*jw.org* <http://www.jw.org>
www.*Slowmersion*.com <http://www.Slowmersion.com>
www.*KneeHowdy*.com <http://www.KneeHowdy.com>
www.*PointlessScore*.com <http://www.PointlessScore.com>
www.NathanCain.com
